autogluon/utils/tabular/ml/trainer/model_presets/presets.py

```diff
@@ -196,3 +213,16 @@ def _add_models(models, options, name_prefix, model_fn):
     else:
         models.append(model_fn(name_prefix, options))
 
+
+def __is_fastai_available():
+    try:
+        with warning_filter():
+            import fastai
```

Comment:
Are there any version number restrictions here?  If so, can you add comment listing the version restrictions

Reply:
added comments